packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -22,14 +22,11 @@ import Button, { ButtonProps } from '../Button';
 type Action = Omit<ButtonProps, 'variant'>;
 
 export interface ButtonGroupProps {
-  /**
-   * Buttons to group.
-   */
   /**
    * @deprecated Use the `actions` prop instead.
    */
   children?:
-    | (ReactElement<ButtonProps> | undefined)[]
+    | (ReactElement<ButtonProps> | null | undefined)[]
```

Comment:
Why can this be `null`? 🤔 are missing children not `undefined`?

Reply:
This is to cover cases like `{somePropThatCanBeNull && <Button />` as children of the ButtonGroup.

Reply:
Ah nice